charts/civil-ccd/values.preview.template.yaml

```diff
@@ -1,5 +1,6 @@
 civil-service:
   java:
+    image: 'hmctspublic.azurecr.io/civil/service:pr-2217'
```

Comment:
is it expected?

Reply:
We will remove this once civil service  merged with master